GitLab/scripts/base.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 BUILD_ID="runner-$CUSTOM_ENV_CI_RUNNER_ID-project-$CUSTOM_ENV_CI_PROJECT_ID-concurrent-$CUSTOM_ENV_CI_CONCURRENT_PROJECT_ID"
-CONNECTION_INFO_ID=$BUILD_ID-connection-info
+export CONNECTION_INFO_ID=$BUILD_ID-connection-info
```

Comment:
Is there a need for this export?

Reply:
The need is to comply with this `shellcheck` rule: https://github.com/koalaman/shellcheck/wiki/SC2034
We could also disable it

Reply:
Got it.
Well, I am ok with both.